koiki/tests/test_client.py

```diff
@@ -90,44 +118,4 @@ def test_create_delivery_sends_request_with_body(self, post_mock, _logger_mock):
 
         Client(self.order, auth_token='xxx').create_delivery()
 
-        post_mock.assert_called_with(
-            'https://testing_host/rekis/api/altaEnvios',
-            json={
-                'token': 'xxx',
-                'formatoEtiqueta': 'PDF',
-                'envios': [
-                    {
-                        'numPedido': self.order['order_key'],
-                        'bultos': 1,
-                        'kilos': 1.0,
-                        'tipoServicio': '',
-                        'reembolso': 0.0,
-                        'observaciones': self.order['customer_note'],
-
-                        'nombreRemi': 'La Zona',
-                        'apellidoRemi': '',
-                        'direccionRemi': 'C/ La Zona, 1',
-                        'numeroCalleRemi': '',
-                        'codPostalRemi': '08186',
-                        'poblacionRemi': 'Barcelona',
-                        'provinciaRemi': 'Barcelona',
-                        'paisRemi': 'ES',
-                        'emailRemi': 'lazona@opcions.org',
-                        'telefonoRemi': '518888191',
-
-                        'nombreDesti': shipping['first_name'],
-                        'apellidoDesti': shipping['last_name'],
-                        'direccionDesti': shipping['address_1'],
-                        'direccionAdicionalDesti': shipping['address_2'],
-                        'numeroCalleDesti': '',
-                        'codPostalDesti': shipping['postcode'],
-                        'poblacionDesti': shipping['city'],
-                        'provinciaDesti': shipping['state'],
-                        'paisDesti': shipping['country'],
-
-                        'telefonoDesti': billing['phone'],
-                        'emailDesti': billing['email'],
-                    }
-                ]
-            }
-        )
```

Comment:
this has proven to be annoying without providing any value. There were always other more meaningful tests failing.